.github/workflows/flow-coverage.yml

```diff
@@ -17,6 +17,7 @@ on:
 env:
   VERBOSE_UTESTS: 1
   REJSON: ${{ inputs.rejson && 1 || 0 }}  # convert the boolean input to numeric
+  COV: 1
```

Comment:
Where is this used/needed?

Reply:
Required so any call to `make` will be with the coverage context. Added a comment
The necessity was shadowed by `readies`